### PR TITLE
feat: Add Kubernetes Action

### PR DESCRIPTION
**Pull request by simar7**
**User Story:**

As a security practitioner, I would like to manage my Kubernetes cluster for security alerts, so that upon receiving Postee event, I add the following:
1. Labels to pods with pre defined values.
2. Annotations to pods with predefined values.

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

**Comment by simar7** [comment]
> @simar7 could you give more context about this PR?, What kind of new features this PR will introduce?

I added a user story to the PR. It's still in draft phase so I didn't update just yet.

File: outputs/kubernetes.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
+package outputs
```
Comment (AndreyLevchenko) [inline outputs/kubernetes.go:1]: @simar7 
you just added `exec` output. Is there any chance you can use it and shell script that calls `kubectl annotate` ?

Comment (simar7) [reply outputs/kubernetes.go:1]: we certainly can - the idea is to promote Kubernetes as a first class citizen. I am with you on the fact that anything can be done with just a shell and an http request :)

Comment (AndreyLevchenko) [reply outputs/kubernetes.go:1]: AquaServer can be integrated with k8s itself :
https://dzone.com/articles/implementing-aqua-security-to-secure-kubernetes
so from app scopes perspective it looks like some overlap that may lead to confusion.

**Review by AndreyLevchenko — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -21,7 +21,7 @@ fmt :
 	$(GO_FMT) -s -w ./
 
 test :
-	go test -race -coverprofile=coverage.txt -covermode=atomic ./router ./msgservice ./dbservice ./formatting ./data ./regoservice ./routes
+	go test -race -v ./...
```
Comment (krol3) [inline Makefile:24]: we could add this params too:  -v -short -race -timeout 30s -coverprofile=coverage.txt ./...

**Pull request merged**